<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTracking</class>
 <widget class="QWidget" name="FormTracking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="formGroupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>451</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="iDTrackingLabel">
      <property name="text">
       <string>ID Tracking</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="iDTrackingLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="noResiLabel">
      <property name="text">
       <string>No Resi</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="noResiLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="iDKurirLabel">
      <property name="text">
       <string>ID Kurir</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="iDKurirLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="tanggalKirimLabel">
      <property name="text">
       <string>Tanggal Kirim </string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDateEdit" name="tanggalKirimDateEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="tanggalSampaiLabel">
      <property name="text">
       <string>Tanggal Sampai</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateEdit" name="tanggalSampaiDateEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="statusKirimLabel">
      <property name="text">
       <string>Status Kirim</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="statusKirimLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="keteranganLabel">
      <property name="text">
       <string>Keterangan</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="keteranganLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>280</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableTracking">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>451</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
